pelican/migration/generate_settings.py

```diff
@@ -179,15 +179,15 @@ def main():
     parser.add_argument('-y', '--yaml', required=True, help="Pelicanconf YAML file")
     args = parser.parse_args()
 
-    pelconf_yaml = args.yaml
-    sourcepath = os.path.dirname(os.path.realpath(pelconf_yaml))
+    pelicanconf_yaml = args.yaml
```

Comment:
there's nothing horribly wrong w/ using short names, but there didn't seem to be a particularly good reason for it either.